web/cobrands/zurich/layout.scss

```diff
@@ -30,9 +30,16 @@ body.mappage {
     #site-header {
         // The header on a map page needs a shadow too
         @include box-shadow(0 0 6px 1px #000);
-        z-index: 2; // One more than #zurich-main-nav so it's on top
+        z-index: auto; // So that #zurich-main-nav rises to the top
```

Comment:
It feels like this change shouldn't be needed (also the comment isn't right, it's so it's not on top, presumably?). Why is the header bigger than the space available to it?

If we look at https://www.stadt-zuerich.ch/portal/de/index.html and compare against a map page here, the bar is basically the same size. If on desktop the header_logobar had padding top/bottom of 20px, the site-logo position was top left rather than centred vertically, and its height was 72px, I think that would look more like the main site, and you wouldn't need to change this line.
The only other change needed would be the 117px would change to 112px in the front page table cell to match the slight change there.